### ISSUE TITLE
Ethernet support for BTT Pi4B?

### ISSUE THREAD

**Issue by bignem** [stas2z/orangepi-build-cb1#1]
Is the ethernet port supposed to work on the Pi4B on this release? It does not appear to be detected on my board. The wifi does not seem to be stable for me, I was able to connect, and ping google with 3 packets before it failed. Rebooting does not seem to help at this time. Maybe a temperature issue? It does feel to be pretty warm.

**Comment by shawnchr** [comment]
They disabled ethernet in Ver 2.1.2 and are still debugging it. 2.1.2 was built to allow wireless users to having a working board.

**Comment by shawnchr** [comment]
Yes this will work on the Pi4B board but you will need to configure the ethernet in nmtui

**Comment by stas2z** [comment]
Yes, eth doesnt work, as it based on zero2 which have external gbit eth. Will try to enable internal 100mb eth to fix this issue. But it will be not so easy as we dont have schematics of the cb1.

Btw, my cb1 wifi is pretty stable, mb smth wrong with antenna

**Comment by frutabruta** [comment]
> Is the ethernet port supposed to work on the Pi4B on this release? It does not appear to be detected on my board. The wifi does not seem to be stable for me, I was able to connect, and ping google with 3 packets before it failed. Rebooting does not seem to help at this time. Maybe a temperature issue? It does feel to be pretty warm.

Some CB1's have faulty wi-fi, those are being replaced. You can apply on FB or Discord. Works fine on my CB1.

**Comment by zhang-benj** [comment]
> Yes this will work on the Pi4B board but you will need to configure the ethernet in nmtui
@shawnchr Can you elaborate? Nmtui doesn't even detect the ethernet for me, running a mantaM8p as the carrier board. Note that ethernet DOES work on other images, but those have bugged wifi and can't be used
